modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-status.adoc

```diff
@@ -1,15 +1,37 @@
 = rpk cluster self-test status
 :description: Reference for the 'rpk cluster self-test status' command. Queries the status of the currently running or last completed self-test run.
 
-Queries the status of the currently running or last completed self-test run. Returns the status of the currently running or last completed self-test run.
+Queries the status of the currently running or last completed self-test run. Returns the status of the currently running or the results of the last completed self-test.
 
 Use this command after invoking xref:./rpk-cluster-self-test-start.adoc[rpk cluster self-test start] to determine the status of
 the jobs launched. Possible results are:
 
 * One or more jobs still running
-** Returns the IDs of Redpanda nodes still running self-tests.
+** Returns the IDs of Redpanda nodes still running self-tests. Example:
+
+[,bash,role=no-copy]
+----
+Node 1 is still running net self test
+----
+
 * No jobs running
-** Returns cached results for all nodes of the last completed test.
+** Returns the cached results for all nodes of the last completed test.
+
+Test results are grouped by node ID. Each test returns the following:
+
+* *Name*: Description of the test.
+* *Info*: Details about the test run attached by Redpanda.
+* *Type*: Either `disk`, `network`, or `cloud` test.
+* *Test Id*: Unique identifier given to jobs of a run. All IDs in a test should match. If they don't match, then newer and/or older test results have been included erroneously.
+* *Timeouts*: Number of timeouts incurred during the test.
+* *Start time*: Time that the test started, in UTC.
+* *End time*: Time that the test ended, in UTC.
+* *Avg Duration*: Duration of the test.
+* *IOPS*: Number of operations per second. For disk, it's `seastar::dma_read` and `seastar::dma_write`. For network, it's `rpc.send()`.
+* *Throughput*: For disk, throughput rate is in bytes per second. For network, throughput rate is in bits per second in. Note that GiB vs. Gib is the correct notation displayed by the UI.
+* *Latency*: 50th, 90th, etc. percentiles of operation latency, reported in microseconds(us). Represented as P50, P90, P99, P999, and MAX respectively.
```

Comment:
```suggestion
* *Latency*: 50th, 90th, etc. percentiles of operation latency, reported in microseconds (μs). Represented as P50, P90, P99, P999, and MAX respectively.
```
Not sure if that will render correctly, but I am assuming that was the intended character